cypress/e2e/home.cy.ts

```diff
@@ -1,8 +1,67 @@
-const projectUrl = Cypress.env("BASE_URL");
+import { projectUrl, stellarIconUrl } from "../support/constants";
 
-describe("Navigation", () => {
-  it("Go to the website:", () => {
-    console.log(projectUrl, "ProjectURL");
-    cy.visit(projectUrl);
+beforeEach(() => {
+  cy.visit(projectUrl);
+});
+
+describe("Home: UI and functionality:", () => {
+  it("Home UI: Stellar logo", () => {
+    cy.get('[data-cy="stellar-home-logo"]').should("be.visible");
+  });
+  it("Home UI: Should show a title and two paragraphs", () => {
+    cy.get('[data-cy="rocket-title"]')
+      .should("be.visible")
+      .and("contain", "Rocket");
+    cy.get('[data-cy="rocket-description-1"]')
+      .should("be.visible")
+      .and("contain", "The easiest way to connect with a wallet");
+    cy.get('[data-cy="rocket-description-2"]')
+      .should("be.visible")
+      .and("contain", "Sign in methods");
+  });
+
+  it("Button connect: Should show a button to generate connect", () => {
+    cy.get('[data-cy="connect"]')
+      .should("be.visible")
+      .and("contain", "Connect with a secret key");
+    cy.get('[data-cy="connect"]').should("exist");
+  });
+
+  it("Button create wallet: Should show a button to generate a pair of new keys", () => {
+    cy.get('[data-cy="register"]').should("exist");
+    cy.get('[data-cy="register"]').should(
+      "have.text",
+      "Generate key pair for a new account"
+    );
+    cy.get('[data-cy="public-key"]').should("not.exist");
+    cy.get('[data-cy="private-key"]').should("not.exist");
+    cy.get('[data-cy="confirm-wallet"]').should("not.exist");
+    cy.get('[data-cy="go-back"]').should("not.exist");
+    cy.get('[data-cy="register"]').click();
+    cy.get('[data-cy="public-key"]').should("exist");
+    cy.get('[data-cy="private-key"]').should("exist");
+    cy.get('[data-cy="confirm-wallet"]').should("exist");
+    cy.get('[data-cy="go-back"]').should("exist");
+    cy.get('[data-cy="confirm-wallet"]').should("have.text", "Confirm wallet");
+    cy.get('[data-cy="go-back"]').should("have.text", "Go back");
+  });
+});
+
+describe(" Navbar: UI and functionality:", () => {
+  it("Check Stellar Icon functionality", () => {
```

Comment:
`Should...`

Reply:
Solved in commit f0a74323da3f19ea9b248e1159579dd2625abb9d.